### DISCUSSION TITLE
[FEATURE]2FA允许使用前一个30秒的验证码

### DISCUSSION THREAD

**Discussion by shutdown-awa** [1Panel-dev/1Panel#5941] [Ideas]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

有时候输入验证码只剩几秒，输完按完确认又刷新了，然后还要重新看一遍验证码。我这种用Bitwarden自动复制验证码的有时候真的很难顶QAQ。

### 请描述你建议的实现方案

比如说验证码30秒更新一次，当用户输入上一个30秒的验证码也允许通过。

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。我们目前不考虑支持使用前 30 秒的验证码进行验证，望理解。
在面板设置中，你可以开启两步验证并设置刷新时间。
针对你的情况，建议适当增加刷新时间。

**Comment by QYG2297248353** [comment]
2FA 不是支持 容错时间么
